dm/syncer/syncer.go

```diff
@@ -2256,24 +2256,9 @@ func (s *Syncer) Run(ctx context.Context) (err error) {
 		var originSQL string // show origin sql when error, only ddl now
 		var err2 error
 		var sourceTable *filter.Table
+		var needContinue bool
 
-		switch ev := e.Event.(type) {
-		case *replication.RotateEvent:
-			err2 = s.handleRotateEvent(ev, ec)
-		case *replication.RowsEvent:
-			eventIndex++
-			s.metricsProxies.Metrics.BinlogEventRowHistogram.Observe(float64(len(ev.Rows)))
-			sourceTable, err2 = s.handleRowsEvent(ev, ec)
-			if sourceTable != nil && err2 == nil && s.cfg.EnableGTID {
-				if _, ok := affectedSourceTables[sourceTable.Schema]; !ok {
-					affectedSourceTables[sourceTable.Schema] = make(map[string]struct{})
-				}
-				affectedSourceTables[sourceTable.Schema][sourceTable.Name] = struct{}{}
-			}
-		case *replication.QueryEvent:
-			originSQL = strings.TrimSpace(string(ev.Query))
-			err2 = s.ddlWorker.HandleQueryEvent(ev, ec, originSQL)
-		case *replication.XIDEvent:
+		funCommit := func() (needContinue bool, err error) {
```

Comment:
```suggestion
		funcCommit := func() (bool, error) {
```
this function doesn't use the named return var `needContinue`